app/views/users/show.html.haml

```diff
@@ -10,37 +10,63 @@
 
 
 .entry-content
-  %p
-    %b #{t('.email')}:
-    = @user.email
-
-  - if @user.last_sign_in_at.blank? && @user.current_sign_in_at.blank?
-    %p #{@user.email} #{t('.user_has_never_signed_in')}
+  = field_or_none("#{t('.email')}", mail_to(@user.email), label_class: 'standard-label')
 
+  - unless @user.membership_applications.exists?
+    = field_or_none("#{t('applications')}", t('none'), label_class: 'standard-label')
   - else
+    -# .col-sm-3{ style: 'margin-left: -15px;' }
+    %strong #{t('applications')}:
```

Comment:
assign a style  & set the style to use 'strong' instead of coding `strong` in here directly.